server/src/main/kotlin/suwayomi/tachidesk/global/impl/AppUpdate.kt

```diff
@@ -22,12 +23,12 @@ data class UpdateDataClass(
     val url: String,
 )
 
-object AppUpdate {
+object AppUpdate : KoinComponent {
     private const val LATEST_STABLE_CHANNEL_URL = "https://api.github.com/repos/Suwayomi/Suwayomi-Server/releases/latest"
     private const val LATEST_PREVIEW_CHANNEL_URL = "https://api.github.com/repos/Suwayomi/Suwayomi-Server-preview/releases/latest"
 
-    private val json: Json by injectLazy()
-    private val network: NetworkHelper by injectLazy()
+    private val json: Json by inject()
+    private val network: NetworkHelper by inject()
```

Comment:
I think its fine to keep using injectLazy, since it allows us to go without KoinComponent. KoinComponent is more for scoping to a context, which we don't need for out application.